### ISSUE TITLE
Issues with FSharp.Core

### ISSUE THREAD

**Issue by sergey-tihon** [sergey-tihon/SwaggerProvider#12]
For me it looks like an issue with version of `FSharp.Core`.

@centur reported in Gitter

> Hi, a bit out of the blue question - I'm trying to run SwaggerProvider in LinqPad using sample code from tutorial and it doesn't work

```
Method not found: 'Microsoft.FSharp.Core.FSharpFunc`2<System.String,Microsoft.FSharp.Core.FSharpFunc`2<System.Type,System.Object>> SwaggerProvider.Int... 

InnerException
null 

TargetSite

6RuntimeMethodInfo4 
Pet.GetPetById (Int64 petId) 

StackTrace
   at Main.PetStore.Pet.GetPetById(Int64 petId)
   at <StartupCode$query_mjdlse>.$Main.main@() in C:\Users\Alexey\AppData\Local\Temp\LINQPad5\_homqyiak\query_mjdlse.fs:line 35
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart() 

HelpLink
null 

HResult
-2146233069 

Message
Method not found: 'Microsoft.FSharp.Core.FSharpFunc`2<System.String,Microsoft.FSharp.Core.FSharpFunc`2<System.Type,System.Object>> SwaggerProvider.Internal.RuntimeHelpers.get_deserialize()'.
```

I know that there is a guide [Notes and Guidance on FSharp.Core](https://fsharp.github.io/2015/04/18/fsharp-core-notes.html)

But what it the best way to handle this in TP?
// @forki could you advise plz?

**Comment by forki** [comment]
I'm not sure

**Comment by sergey-tihon** [comment]
Should be fixed in `v0.3.2`
